components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java

```diff
@@ -343,8 +343,12 @@ public OAuthRevocationResponseDTO revokeTokenByOAuthClient(OAuthRevocationReques
                     }
 
                 } else {
-
-                    accessTokenDO = OAuth2Util.findAccessToken(revokeRequestDTO.getToken(), true);
+                    try {
+                        accessTokenDO = OAuth2Util.findAccessToken(revokeRequestDTO.getToken(), true);
```

Comment:
Ideally, this method should not through IllegalArgumentException. Isn't it?
Shouldn't we try to fix is there?